color_min : [ 0 , 250 , 0]
color_max : [5 , 255 , 1]
camera_z : 40
box_z : 0.1